packages/google_sign_in/google_sign_in_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.0.2
```

Comment:
Nit: since you are adding a feature these should be minor version bumps rather than bugfix version bumps.